src/ui/routes/+layout.svelte

```diff
@@ -18,26 +18,15 @@
 	}
 </script>
 
-<section>
-	<img alt="Zarf logo" id="logo" src={logo} />
+<Header />
 
+<!-- <section>
```

Comment:
Don't comment code out, just kill it with fire, that's what SCM is for.

Reply:
🔥📕